website/docs/language/expressions/operators.mdx

```diff
@@ -42,6 +42,8 @@ given values to be of a particular type. Terraform will attempt to convert
 values to the required type automatically, or will produce an error message
 if automatic conversion is impossible.
 
+**Note**: The `?` symbol, when combined with `:`, is part of a conditional expression in Terraform and not considered an operator. For more informaton on how to use conditional expressions, see [Conditional Expressions](/terraform/language/expressions/conditionals#conditional-expressions).
```

Comment:
```suggestion
The `?` character combined with the `:` character is part of a conditional expression in Terraform and is not considered an operator. For more information, refer to [Conditional Expressions](/terraform/language/expressions/conditionals#conditional-expressions).
```

I know I changed a lot of _little_ things, so I wanted to provide the context on the style guide rules of why I made this suggestion:
1. We try to [avoid callouts](https://github.com/hashicorp/engineering-docs/blob/main/writing/style-guide.md#callout-boxes) unless they are absolutely necessary, so I feel like we can remove the call out here.
2. We use "refer to" instead of "see" as it's more accessible for those using screen readers.